File: src/main/java/project/rpg/commands/LevelCommand.java
package project.rpg.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import org.jetbrains.annotations.NotNull;

import static project.rpg.player.Human._playerMap;

public class LevelCommand implements CommandExecutor {

    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
        if (sender instanceof Player) {
            Player player = (Player) sender;

            if (args.length == 2) {
                switch (args[0]) {
                    case "add":
                        _playerMap.get(player.getUniqueId()).getLevel().addExp(Integer.parseInt(args[1]));
                        break;
                }
            }
        }
        return false;
    }
}

File: src/main/kotlin/project/rpg/commands/LevelCommand.kt
package project.rpg.commands

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.command.TabCompleter
import org.bukkit.entity.Player
import project.rpg.player.User.Companion.getPlayer

class LevelCommand : CommandExecutor, TabCompleter {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if (sender is Player) {
            if (args.size == 2) {
                when (args[0]) {
                    "add" -> getPlayer(sender)!!.levels.addExp(args[1].toLong())
                    "remove" -> getPlayer(sender)!!.levels.addExp(-args[1].toLong())
                    else -> {}
                }
            }
        }
        return false
    }

    override fun onTabComplete(
        p0: CommandSender,
        p1: Command,
        p2: String,
        p3: Array<out String>?
    ): List<String> {
        return emptyList()
        TODO("Not yet implemented")
    }
}
